lib/libc/stdlib/exit.3

```diff
@@ -78,7 +78,7 @@ The
 implementation of the
 .Fn _Exit
 function does not call destructors registered with
-.Xr __cxa_atexit 3,
+.Xr __cxa_atexit 3 ,
```

Comment:
this man page doesn't exist tho

(at least i can't find it on https://man.freebsd.org/ or https://man-dev.freebsd.org/)

Reply:
This exists at: https://man.freebsd.org/cgi/man.cgi?query=exit&apropos=0&sektion=3&manpath=FreeBSD+15.0-CURRENT&arch=default&format=html

It's a libc function.

Reply:
I mean the man page explaining `__cxa_atexit`

Reply:
I believe __cxa_atexit() is an internal function and not something that can be used by user-space programs.

But I'm not sure why there's no man page.